src/modules/core/index.ts

```diff
@@ -19,10 +19,12 @@ import { ServiceWorkerConfiguration } from '../../configuration';
 import { AssetCachingOptions } from '../asset-caching';
 import { LinkPrefetchOptions } from '../link-prefetch';
 import { OfflinePageOptions } from '../offline-page';
+
 declare global {
   interface WorkerGlobalScope {
     AMP_SW: {
       init: Function;
+      installNoOpServiceWorker: Function;
```

Comment:
Can we add more explicit typing for what the `installNoOpServiceWorker` function is?